v1/yang/aft/openconfig-aft-network-instance.yang

```diff
@@ -0,0 +1,182 @@
+module openconfig-aft-network-instance {
```

Comment:
Do we need this one in the repo? It's augmenting `/network-instances` that doesn't exist in gRIBI.

Reply:
Done